### ISSUE TITLE
Escaping of spaces in node/edge names

### ISSUE THREAD

**Issue by Baschdl** [KDD-OpenSource/32de-python#5]
Implement escaping or removing of spaces in node and edge names in `to_str` function.

**Comment by feeds** [comment]
is this only about spaces? 
e.g. "writes book" sould be "writes_book" or "writes book"?

**Comment by Baschdl** [comment]
> is this only about spaces?
The reason is that the edge and node types should be recognized as "words" by the tf-idf-Vectorizer from sklearn.

**Comment by jexp** [comment]
In cypher you can use backticks around node-labels and property-key and rel-types.

In the java/kernel API you just use Java Strings.

**Comment by feeds** [comment]
apparently you can override the word tokenizer with your own callable, so maybe use
`tokenizer=lambda sequence: sequence.split(';')` and we can just pass the metapath with ';' as a delimter
